### DISCUSSION TITLE
Styling of (Single) Checkbox

### DISCUSSION THREAD

**Discussion by MichaelSchmidle** [formkit/formkit#1321] [Ideas]
The Regenesis style of a single checkbox currently differs a bit from other form elements. The **label is not bold** and the **input's border/ring doesn't turn to the error color**. It looks like that:

![Screen Shot 2024-04-21 at 10 00 19](https://github.com/formkit/formkit/assets/552446/44892435-be64-4c6b-a200-ba2bac70ebda)

To bring this more in line with the rest of the styles, we would need to change `"family:box__label"` and `"family:box__decorator"` inside `formkit.theme.ts` respectively as follows:

```
"!font-normal": false,
```

```
"group-data-[invalid]:border-red-400": true,
"group-data-[invalid]:ring-1": true,
"group-data-[invalid]:ring-red-400": true,
```

After adding this to `formkit.theme.ts`, the checkbox now looks like this with a bold label and the error color of the input – consistent with the rest of the theme:

![Screen Shot 2024-04-21 at 09 56 45](https://github.com/formkit/formkit/assets/552446/de9eedcf-6589-4521-a9f3-ecc623ab40e0)

What do you think, could this be addressed in the Regenesis theme generator? Note: For multi-checkboxes, the case might be different.

**Comment by fenilli** [comment]
I don't know if I agree with the bold, but the border being red is indeed a really good idea @andrew-boyd.
